tests/unit/dojson/fixtures/test_hep_record.xml

```diff
@@ -197,6 +197,21 @@
     <subfield code="o">ConfAcr</subfield>
     <subfield code="0">1</subfield>
   </datafield>
+  <datafield tag="773" ind1="1" ind2=" ">
+    <subfield code="c">R123</subfield>
+    <subfield code="n">2</subfield>
+    <subfield code="p">Phys.Rev.Lett.</subfield>
+    <subfield code="v">105</subfield>
+    <subfield code="y">2010</subfield>
+    <subfield code="m">note</subfield>
+    <subfield code="z">isbn</subfield>
+    <subfield code="x">pubinfo_freetext</subfield>
+    <subfield code="w">cnum</subfield>
+    <subfield code="t">confpaper_info</subfield>
+    <subfield code="r">reportNumber</subfield>
+    <subfield code="o">ConfAcr</subfield>
+    <subfield code="0">1</subfield>
+  </datafield>
```

Comment:
I was about to change it, but here I believe it has a value in its form, because it test also it's interaction WRT the non hidden publication in the same record, and it very practically allows to test the json->marc transformation as well.